test/known_issues/known_issues.status

```diff
@@ -9,6 +9,7 @@ prefix known_issues
 [$system==win32]
 test-stdout-buffer-flush-on-exit: SKIP
 test-cluster-disconnect-handles: SKIP
+test-dgram-bind-shared-ports-after-port-0: SKIP
```

Comment:
Apparently windows does not support `process.send` for cluster workers, but since is a `known_issue` I let it fail